Tools/sitl_run.sh

```diff
@@ -41,8 +43,8 @@ if [ "$model" == "" ] || [ "$model" == "none" ]; then
 	model="iris"
 fi
 
-if [ "$#" -lt 6 ]; then
-	echo usage: sitl_run.sh sitl_bin debugger program model src_path build_path
+if [ "$#" -lt 7 ]; then
+	echo usage: sitl_run.sh sitl_bin debugger program model world src_path build_path
```

Comment:
Although, given there is a check anyway, it should be fine.

Reply:
Right...took me a while to realize this though :)